pkg/services/object/acl/basic_helper.go

```diff
@@ -121,6 +121,21 @@ func (a basicACLHelper) SystemAllowed(op eacl.Operation) bool {
 	return false
 }
 
+// InnerRingAllowed returns true if inner ring node as a subset of system group
+// allowed the n-th operation is set in ACL.
+func (a basicACLHelper) InnerRingAllowed(op eacl.Operation) bool {
+	switch op {
+	case eacl.OperationSearch, eacl.OperationRangeHash, eacl.OperationHead:
+		return true
+	default:
+		if n, ok := order[op]; ok {
```

Comment:
Can we use `SystemAllowed` here?

Reply:
`SystemAllowed` allows `Put` and `Get` request for private containers, unfortunately.